### PR TITLE
cancancan suggestions

### PR DESCRIPTION
**Pull request by jmartin-sul**
it wasn't really any harder to implement my suggestions for the cancancan branch than it was to make the comments, so i went ahead and created this PR.  hoping these suggestions are uncontroversial, but just in case, i tried to make them as separate commits that can be cherry picked or squashed as desired.

@cbeer @darrenleeweber

### CODE REVIEW

File: app/views/catalog/_ds.html.erb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <%
-  apo = @obj.admin_policy_object
```
Comment (jmartin-sul) [inline app/views/catalog/_ds.html.erb]: this was no longer used, AFAICT

File: app/models/ability.rb [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+#      * the :manage_content rule would be checked using the logic for ActiveFedora::Base before falling through
+#      to the Dor::AdminPolicyObject logic.
+#    * more info at https://github.com/CanCanCommunity/cancancan/wiki/Ability-Precedence
+#  * Argo uses this pattern for checking permissions via CanCan:  https://github.com/CanCanCommunity/cancancan/wiki/Non-RESTful-Controllers
+#
+# Argo specific notes:
+#   * permissions granted by an APO apply to objects governed by the APO, as well as to the APO itself; hence, the fallthrough
+#   check to the APO's own ID for Dor::AdminPolicyObject types in addition to the checks for ActiveFedora::Base types (e.g. the
+#   "can :manage_item, Dor::AdminPolicyObject" and "can :manage_item, ActiveFedora::Base" definitions, with the latter being checked
+#   first).  see https://github.com/sul-dlss/argo/issues/76
```
Comment (jmartin-sul) [inline app/models/ability.rb]: seemed like it'd be nice to have an intro for people new to cancan

File: spec/models/ability_spec.rb [hunk 3]
```diff
@@ -55,8 +56,8 @@
```
Comment (jmartin-sul) [inline spec/models/ability_spec.rb]: rationales for all the below renamed identifiers in the commit msg

File: spec/models/ability_spec.rb [hunk 4]
```diff
@@ -35,6 +35,7 @@
     it { should_not be_able_to(:manage_item, item) }
     it { should_not be_able_to(:manage_content, item) }
     it { should_not be_able_to(:manage_desc_metadata, item) }
+    it { should_not be_able_to(:create, Dor::AdminPolicyObject) }
```
Comment (jmartin-sul) [inline spec/models/ability_spec.rb]: seemed like a reasonable expectation to place on a viewer

**Comment by cbeer** [comment]
:+1:

**Pull request merged**